marktplaats/models/listing_image.py

```diff
@@ -21,3 +31,31 @@ def parse(cls, data):
                 image_data["extraExtraLargeUrl"],
             ))
         return images
+
+
+def fetch_listing_images(listing_id: str) -> list[str]:
+    """
+    Returns a list of image URLs for a given listing.
+    It scrapes the listing page and parses the ld+json objects on that page.
+    :param listing_id: The listing ID to get images for.
+    :return: A list of image URLs (https).
+    """
+    r = requests.get(
+        f"https://link.marktplaats.nl/{listing_id}",
+        headers=REQUEST_HEADERS)
```

Comment:
Nit: this isn't PEP8 compliant
```suggestion
        headers=REQUEST_HEADERS,
    )
```